### PR TITLE
Update product website, fix typo in UG edit command.

### LINKED ISSUE #71: Update product website
Our [product website's](https://ay2021s1-cs2103t-f13-2.github.io/tp/) main page is currently not updated. The main page should probably be updated to have the same content as this repo's README.  The site is being built from the `docs` folder. The files to change would probably be `docs/index.md` which holds the content to be displayed on the main page, and `docs/_config.yml` to change the title from **AB-3** to **bookmark**.

### PR DESCRIPTION
**Pull request by pangpuncake**
Closes #71

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -365,22 +363,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
+* 2a. The list is empty.
+
```
Comment (TanLeYang) [inline docs/DeveloperGuide.md:367]: I think this extension 2a isn't supposed to be here anymore, its included in UC05 already.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -365,22 +363,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    Use case ends.
+
 * 3a. The index given is invalid
-    
+
     * 3a1. bookmark shows an error message
-
```
Comment (TanLeYang) [inline docs/DeveloperGuide.md:371]: Just saw that the extension numbering here is incorrect, should be 2a, 2a1, 2b etc. Can you help change them too 🙏 thanks!

**Review by pennhanlee — COMMENTED** [review]
LGTM

**Review by TanLeYang — APPROVED** [review]
LGTM!

**Pull request merged**